heron/schedulers/src/java/com/twitter/heron/scheduler/yarn/HeronExecutorTask.java

```diff
@@ -133,6 +133,7 @@ public HeronExecutorTask(final REEFFileNames fileNames,
         env,
         topologyJar,
         topologyDefFile,
+        false,
```

Comment:
if it's hard to propagate verbose flag to YARN scheduler, I'm not sure what to do. Maybe just put `false` here?